scripts/scripts.js

```diff
@@ -22,9 +22,8 @@ const AUTHORS = {
 };
 
 export const ARTICLE_TEMPLATES = {
```

Comment:
Maybe we should rename this to just TEMPLATES wdyt ?

Reply:
I renamed it